tests/rule_based_profiler/config/test_base.py

```diff
@@ -222,8 +216,10 @@ def test_rule_based_profiler_config_successfully_loads_with_required_args():
     }
     schema = RuleBasedProfilerConfigSchema()
     config = schema.load(data)
-    assert isinstance(config, RuleBasedProfilerConfig)
-    assert len(config.rules) == 1 and isinstance(config.rules["rule_1"], RuleConfig)
+    assert isinstance(config, dict)
```

Comment:
aha this is because we're using `Schema` and not `NotNullSchema` right? The `@post_load` hook would normally convert this!

I'm going to close my earlier comments around this.